src/omv/ports/stm32/sensor.c

```diff
@@ -1433,38 +1488,38 @@ void DCMI_DMAConvCpltUser(uint32_t addr)
 }
 
 #if (OMV_ENABLE_SENSOR_MDMA == 1)
-static void mdma_config(MDMA_InitTypeDef *init, sensor_t *sensor, uint32_t bytes_per_pixel)
+static void mdma_config(uint32_t bytes_per_pixel, vbuffer_t *buffer)
```

Comment:
```suggestion
 static void mdma_config(MDMA_InitTypeDef *init, sensor_t *sensor, uint32_t bytes_per_pixel)
```
Please revert mdma_config, it's better to accept these args than to hard-code them, at least results in less changes.

Reply:
Reverted